bin/run-integration-tests.sh

```diff
@@ -5,7 +5,7 @@ set -e
 VINYLDNS_VERSION="0.10.4"
 
 echo "Removing VinylDNS API if exists.."
-if docker ps | grep -q "vinyldns-js-api"; then
+if docker ps -a| grep -q "vinyldns-js-api"; then
```

Comment:
I think the `-a` is added for testing? If so, we can remove them.

Reply:
this done purposely to pull all running docker container and stop which has the name `vinyldns-js-api`